synapse/storage/databases/main/relations.py

```diff
@@ -153,6 +153,12 @@ def _get_recent_references_for_event_txn(
             # If there are more events, generate the next pagination key.
             next_token = None
             if len(events) > limit and last_topo_id and last_stream_id:
```

Comment:
We do we need `last_topo_id` to be truthy here?

Reply:
I think that was for mypy.

Reply:
It was, I've changed them to asserts though since I think that is clearer!